platform/android/export/export.cpp

```diff
@@ -2297,15 +2297,18 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 		if (use_custom_build) {
 			//test that installed build version is alright
-			FileAccessRef f = FileAccess::open("res://android/.build_version", FileAccess::READ);
-			if (!f) {
-				EditorNode::get_singleton()->show_warning(TTR("Trying to build from a custom built template, but no version info for it exists. Please reinstall from the 'Project' menu."));
-				return ERR_UNCONFIGURED;
-			}
-			String version = f->get_line().strip_edges();
-			if (version != VERSION_FULL_CONFIG) {
-				EditorNode::get_singleton()->show_warning(vformat(TTR("Android build version mismatch:\n   Template installed: %s\n   Godot Version: %s\nPlease reinstall Android build template from 'Project' menu."), version, VERSION_FULL_CONFIG));
-				return ERR_UNCONFIGURED;
+			{
+				FileAccessRef f = FileAccess::open("res://android/.build_version", FileAccess::READ);
+				if (!f) {
+					EditorNode::get_singleton()->show_warning(TTR("Trying to build from a custom built template, but no version info for it exists. Please reinstall from the 'Project' menu."));
+					return ERR_UNCONFIGURED;
+				}
+				String version = f->get_line().strip_edges();
+				f->close();
```

Comment:
I think `FileAccessRef` is meant to automatically close the handle when it goes out of scope, so now that you've added an extra scope, the manual `close()` might not be needed. (That being said it likely doesn't hurt either.)

Reply:
From what I can see, `FileAccessRef` only deletes `FileAccess`, it doesn't close it. And I can confirm (at least on 3.2 branch) that `.build_version` was still locked after the export process.

Reply:
Ah my bad, there is `close()` in the specific Windows/Unix implementation of `FileAccess`. I'm not sure why the file was still locked after the export though.